test/Serilog.Enrichers.Sensitive.Tests.Unit/WhenMaskingPaths.cs

```
@@ -0,0 +1,39 @@
+using FluentAssertions;
+using Xunit;
+
+namespace Serilog.Enrichers.Sensitive.Tests.Unit;
+
+public class WhenMaskingPaths
+{
+    private const string Mask = @"***\";
+
+    [Theory]
+    [InlineData(@"C:\Users\Admin\Secret\File.dll", @"***\", false)]
```

I would expect this to be _only_ the mask in the expected output. Right now it looks like you're keeping the directory separator which seems odd.

---

Don't really understand what do you mean

---

I now see what happens here and it was a bit non-obvious.
I would have expected this result to be fully masked, for example `***`. However I missed that you define the _mask value_ as `***\` so it's not the implementation that threw me off here but the default mask value I missed above.